libraries/SPI/SPI.cpp

```diff
@@ -311,53 +319,55 @@ void SPIClass::write32(uint32_t data, bool msb) {
 
 /**
  * Note:
- *  data need to be aligned to 32Bit
- *  or you get an Fatal exception (9)
+ *  data needs to be aligned to 32 bit boundaries like this: char outBuffer[spiBufferSize] __attribute__ ((aligned (4)));
+ *  if not you may get an Fatal exception (9)
  * @param data uint8_t *
  * @param size uint32_t
  */
 void SPIClass::writeBytes(uint8_t * data, uint32_t size) {
+	uint32_t localSize;
```

Comment:
That's true, I'll beautify it. Seems that my editor indeed has used sometimes tabs, sometimes spaces. I've already removed the tabs in my local files, will submit them ASAP.